### PR TITLE
Update: added README templates

### PR DESCRIPTION
**Pull request by rhoads-zach**
@gastaldi @ALRubinger @quintesse  @edewit mind taking a look?

### CODE REVIEW

**Comment by rhoads-zach** [comment]
thanks @gastaldi, merging.

**Comment by rhoads-zach** [comment]
merged: https://github.com/openshiftio/appdev-documentation/commit/c0f3702ef752dfd9eed5739ef7b5b2905da63b3e

File: docs/topics/crud-README.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+:guideURL: http://appdev.openshift.io/docs/wf-swarm-runtime.html
+:fileLocation: /src/main/webapp/index.html
+endif::[]
+
+
+
+= ${mission} - ${runtime} Booster
+
+
+IMPORTANT: While you can run and interact with this booster on our local host, it requires that you also have a database installed and configured. This booster runs best when deployed on OpenShift with a PostGres database. For more details on using this booster with a single-node OpenShift cluster, CI/CD deployments, as well as the rest of the runtime, see the link:{guideURL}[${runtime} Runtime Guide].
```
Comment (Ladicek) [inline docs/topics/crud-README.adoc]: PostGres -> PostgreSQL

File: docs/topics/rest-http-README.adoc [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+$ git clone git@github.com:${loggedUser}/${targetRepository}
+
+$ cd ${targetRepository}
+
+$ {localRunCMD}
+----
+
+== Interact with this Booster Locally
+
+To interact with your booster while its running, use the form at `http://localhost:8080` or the `curl` command:
```
Comment (Ladicek) [inline docs/topics/rest-http-README.adoc]: its -> it's or it is

Also "while it's running" -> "while it's running locally"

File: docs/topics/rest-http-README.adoc [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+To update your booster:
+
+. Stop your booster.
+. Make your change (e.g. edit link:https://github.com/${loggedUser}/${targetRepository}/edit/master/{fileLocation}[`{fileLocation}`]).
+. Restart your booster.
+. Confirm your change appears.
+
+
+== Run this Booster on a Single-node OpenShift Cluster
+If you have a single-node OpenShift cluster, such as Minishift or the Red Hat Container Development Kit, link:http://appdev.openshift.io/docs/minishift-installation.html[installed and running], you can also deploy your booster there. A single-node OpenShift cluster provides you with access to a cloud environment that is similar to a production environment than simply running directly on your local host.
```
Comment (Ladicek) [inline docs/topics/rest-http-README.adoc]: Extra comma in "Minishift or the Red Hat Container Development Kit, installed and running".

"access to a cloud environment that is similar to a production environment than simply running directly on your local host" sounds weird. Maybe something like "access to a cloud-like environment that is more similar to a production environment than simply running directly on your local host"?

**Comment by Ladicek** [comment]
Sorry for reviewing so late, hope that my comments make sense :-)

**Pull request closed without merging**